docker-compose.yml

```diff
@@ -679,6 +680,45 @@ services:
       UBUNTU:
     command: *cpp-command
 
+  ubuntu-cpp-emscripten-python:
```

Comment:
If we want to use Ubuntu based image for this, we should use `ubuntu-python-emscripten`.
If we need to use conda, we should use `conda-python-emscripten`.

Reply:
Could you update this?

Reply:
done it hopefully.